.github/workflows/build.yml

```diff
@@ -0,0 +1,101 @@
+{
+    "name": "Build",
+    "on": {
+      "push": {
+          "branches": [ "master" ]
+      },
+      "pull_request": null,
+    },
+    "jobs": {
+        "linux": {
+            "name": "CI on x86_64",
+            "runs-on": "ubuntu-latest",
+            "strategy": {
+                "fail-fast": false,
+                "matrix": {
+                    "name": [
+                        "python-36",
+                        "python-37",
+                        "python-38",
```

Comment:
These are gonna be challenging to keep working. Is there really an expectation to keep this working on python 3.6?

Reply:
eh I just copied from jwcrypto, it is easy to drop them when they stop working.

The problem I have now is that testing on my branch, actions fail for varius reason. One annoying is that doc fails because ipa stuff can't be found and therefore sphinx auto sections fail.

so far have not been able to make it conditional
I will probably temporarily dro pthe ipa plugin docs

Reply:
A more annoying part is that I get this error:
>       self.assert_ssl_error_msg(["Connection reset by peer",
                                   "SSLV3_ALERT_HANDSHAKE_FAILURE"],
                                  e.exception)

but in the last run only with py37 and py310 ...